### ISSUE TITLE
Bugs_non-desired spine status reset

### ISSUE THREAD

**Issue by Ququkiwi** [aramot/spinytiny#4]
After drawing spine ROIs and first-time saving, the first round of "Recover ROIs" will set every newly saved spine ROI as "eliminated".

**Comment by Ququkiwi** [comment]
Current solution is to remark as active and save again

**Comment by aramot** [comment]
I tried it and so far it looks ok, as long as you always work according to the required order - 
1. Draw Spine
2. Background
3. Dendrite
4. ClearROIs
5. Recover ROIs
I will try it some more

**Comment by Ququkiwi** [comment]
Yep, I followed the same routine, bigger chance that the bug will happen if going through several rounds, sounds like the matlab storing some residual memory
